src/Runner.Worker/Handlers/CompositeActionHandler.cs

```diff
@@ -26,65 +26,60 @@ public sealed class CompositeActionHandler : Handler, ICompositeActionHandler
 
         public async Task RunAsync(ActionRunStage stage)
         {
-            // Validate args.
+            // Validate args
             Trace.Entering();
             ArgUtil.NotNull(ExecutionContext, nameof(ExecutionContext));
             ArgUtil.NotNull(Inputs, nameof(Inputs));
             ArgUtil.NotNull(Data.Steps, nameof(Data.Steps));
 
-            // Resolve action steps
-            var actionSteps = Data.Steps;
-
-            // Create Context Data to reuse for each composite action step
-            var inputsData = new DictionaryContextData();
-            foreach (var i in Inputs)
-            {
-                inputsData[i.Key] = new StringContextData(i.Value);
-            }
-
-            // Initialize Composite Steps List of Steps
-            var compositeSteps = new List<IStep>();
```

Comment:
Changed this variable name to `nestedSteps`. From a certain perspective, could be viewed as a misnomer before. Changed the name to draw a better distinction.